<!DOCTYPE html>
<html>
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" href="resources/primer.css" media="screen"/>
<link rel="stylesheet" href="resources/rec.css" media="screen"/>
<link rel="stylesheet" href="resources/extra.css" media="screen"/>
<link rel="stylesheet" href="resources/owl.css" media="screen"/>
<link rel="stylesheet" href="resources/dark.css" media="(prefers-color-scheme: dark)"/>
<link rel="stylesheet" href="resources/light.css" media="(prefers-color-scheme: light)"/>
<link rel="stylesheet" href="resources/slider.css" media="screen"/>
<meta name="color-scheme" content="dark light">
<script type="module" src="resources/dark-mode-toggle.mjs"></script><div class="darkmode">
	<dark-mode-toggle class="slider"></dark-mode-toggle>
</div>
<link rel="icon" type="image/png" href="resources/rdf.icon"/> <title>Gene expression (GenEx) ontology</title>


<!-- SCHEMA.ORG METADATA -->
<script type="application/ld+json">{"@context":"https://schema.org","@type":"TechArticle","url":"http://purl.org/genex#","image":"http://vowl.visualdataweb.org/webvowl/#iri=http://purl.org/genex#","name":"Gene expression (GenEx) ontology", "headline":"Expression patterns are a major feature of gene description, and are captured in many expression related databases (e.g., Bgee, Expression Atlas, GENEVESTIGATOR, or Tissue). Yet comparison and interoperability among these heterogeneous data sources or providers is limited by a lack of agreement for describing knowledge of gene expression. Ontologies are recognized as an interesting approach to achieve a consensus for knowledge representation, and allow to leverage semantic interoperability, but are lacking for the domain of gene expression. The Relational Ontology (RO) contains preliminary efforts to define terms for describing this domain. Notably, it contains `expressed in` and `expresses` properties (i.e., relations). Nonetheless, further work was necessary to address the following issue: the representation of additional information related to gene expression, such as developmental stages, as well as absent in and highly expressed relations. Furthermore, we reuse parts of the data schemas of the ORTH and UniProt core ontologies to provide the capacity to interoperate with other biological databases from different knowledge domains which are still relevant to the gene expression domain. Moreover, different gene expression databases use distinct confidence scores to assert expressed in or absent in relations. Finally, we also consider during the GenEx development the cost of materialising all gene expression information and how to mitigate this issue. For example, just for Homo sapiens, we can derive from the Bgee database hundreds of millions of expressed in / absent in assertions (i.e., triples).", "dateReleased":"2019-01-25", "dateModified":"2024-04-30", "version":"0.2", "license":"https://creativecommons.org/publicdomain/zero/1.0/", "codeRepository":"https://github.com/biosoda/genex", "author":[{"@type":"Person","name":"Tarcisio Mendes de Farias","url":"https://orcid.org/0000-0002-3175-5372"}], "contributor":[{"@type":"Person","name":"Christophe Dessimoz","url":"https://orcid.org/0000-0002-2170-853X"},{"@type":"Person","name":"Julien Wollbrett","url":"https://orcid.org/0000-0002-3099-3117"},{"@type":"Person","name":"Tarcisio Mendes de Farias","url":"https://orcid.org/0000-0002-3175-5372"},{"@type":"Person","name":"Marc Robinson-Rechavi","url":"https://orcid.org/0000-0002-3437-3329"},{"@type":"Person","name":"Frederic Bastian","url":"https://orcid.org/0000-0002-9415-5104"}]}</script>

<script src="resources/jquery.js"></script> 
<script src="resources/marked.min.js"></script> 
    <script> 
function loadHash() {
  jQuery(".markdown").each(function(el){jQuery(this).after(marked.parse(jQuery(this).text())).remove()});
	var hash = location.hash;
	if($(hash).offset()!=null){
	  $('html, body').animate({scrollTop: $(hash).offset().top}, 0);
}
	loadTOC();
}
function loadTOC(){
	//process toc dynamically
	  var t='<h2>Table of contents</h2><ul>';i = 1;j=0;
	  jQuery(".list").each(function(){
		if(jQuery(this).is('h2')){
			if(j>0){
				t+='</ul>';
				j=0;
			}
			t+= '<li>'+i+'. <a href=#'+ jQuery(this).attr('id')+'>'+ jQuery(this).ignore("span").text()+'</a></li>';
			i++;
		}
		if(jQuery(this).is('h3')){
			if(j==0){
				t+='<ul>';
			}
			j++;
			t+= '<li>'+(i-1)+'.'+j+'. '+'<a href=#'+ jQuery(this).attr('id')+'>'+ jQuery(this).ignore("span").text()+'</a></li>';
		}
	  });
	  t+='</ul>';
	  $("#toc").html(t); 
}
 $.fn.ignore = function(sel){
        return this.clone().find(sel||">*").remove().end();
 };    $(function(){
      $("#abstract").load("sections/abstract-en.html"); 
      $("#introduction").load("sections/introduction-en.html"); 
      $("#nstable").load("sections/ns-en.html"); 
      $("#overview").load("sections/overview-en.html"); 
      $("#description").load("sections/description-en.html"); 
      $("#references").load("sections/references-en.html"); 
      $("#crossref").load("sections/crossref-en.html", null, loadHash); 
    });
    </script> 
  </head> 

<body>
<div class="container">
<div class="head">
<div style="float:right">language <a href="index-en.html"><b>en</b></a> </div>
<h1>Gene expression (GenEx) ontology</h1>
<h2>Release: 2019-01-25</h2>


<dl>
<dt>Modified on: 2024-04-30</dt>
<dt>This version:</dt>
<dd><a href="http://purl.org/genex/0.2">http://purl.org/genex/0.2</a></dd>
<dt>Latest version:</dt>
<dd><a href="http://purl.org/genex#">http://purl.org/genex#</a></dd>
<dt>Revision:</dt>
<dd>0.2</dd>
<dt>Authors:</dt>
<dd><a href="https://orcid.org/0000-0002-3175-5372">Tarcisio Mendes de Farias</a>, <a href="https://www.sib.swiss">https://www.sib.swiss</a></dd>

<dt>Contributors:</dt>
<dd><a href="https://orcid.org/0000-0002-2170-853X">Christophe Dessimoz</a>, <a href="https://www.sib.swiss">https://www.sib.swiss</a></dd><dd><a href="https://orcid.org/0000-0002-3099-3117">Julien Wollbrett</a>, <a href="https://www.sib.swiss">https://www.sib.swiss</a></dd><dd><a href="https://orcid.org/0000-0002-3175-5372">Tarcisio Mendes de Farias</a>, <a href="https://www.sib.swiss">https://www.sib.swiss</a></dd><dd><a href="https://orcid.org/0000-0002-3437-3329">Marc Robinson-Rechavi</a>, <a href="https://www.sib.swiss">https://www.sib.swiss</a></dd><dd><a href="https://orcid.org/0000-0002-9415-5104">Frederic Bastian</a>, <a href="https://www.sib.swiss">https://www.sib.swiss</a></dd>

<dt>Publisher:</dt>
<dd><a href="https://www.sib.swiss">https://www.sib.swiss</a></dd>
<dt>Imported Ontologies:</dt>
<dd><a href="http://purl.obolibrary.org/obo/caro.owl">caro.owl</a></dd><dd><a href="http://purl.obolibrary.org/obo/iao.owl">iao.owl</a></dd><dd><a href="http://purl.obolibrary.org/obo/ro.owl">ro.owl</a></dd><dd><a href="http://purl.org/lscr">lscr</a></dd><dd><a href="http://purl.org/net/orth">orth</a></dd><dd><a href="http://www.ebi.ac.uk/efo/efo.owl">efo.owl</a></dd><dd><a href="http://www.w3.org/2004/02/skos/core">core</a></dd>

<dt>Extended Ontologies:</dt>
<dd><a href="http://purl.obolibrary.org/obo/ro.owl">http://purl.obolibrary.org/obo/ro.owl</a></dd>

<dt>See also:</dt>
<dd><a href="http://purl.obolibrary.org/obo/ro.owl">http://purl.obolibrary.org/obo/ro.owl</a></dd><dd><a href="https://bioschemas.org">https://bioschemas.org</a></dd><dd><a href="https://schema.org">https://schema.org</a></dd>

<dt>Download serialization:</dt><dd><span><a href="ontology.jsonld" target="_blank"><img src="https://img.shields.io/badge/Format-JSON_LD-blue.svg" alt="JSON-LD" /></a> </span><span><a href="ontology.owl" target="_blank"><img src="https://img.shields.io/badge/Format-RDF/XML-blue.svg" alt="RDF/XML" /></a> </span><span><a href="ontology.nt" target="_blank"><img src="https://img.shields.io/badge/Format-N_Triples-blue.svg" alt="N-Triples" /></a> </span><span><a href="ontology.ttl" target="_blank"><img src="https://img.shields.io/badge/Format-TTL-blue.svg" alt="TTL" /></a> </span></dd><dt>License:</dt><dd><a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank"><img src="https://img.shields.io/badge/License-https://creativecommons.org/publicdomain/zero/1.0/-blue.svg" alt="https://creativecommons.org/publicdomain/zero/1.0/" /></a>
</dd><dt>Visualization:</dt><dd><a href="webvowl/index.html#" target="_blank"><img src="https://img.shields.io/badge/Visualize_with-WebVowl-blue.svg" alt="Visualize with WebVowl" /></a></dd>
<!-- <dt>Evaluation:</dt><dd><a href="OOPSevaluation/oopsEval.html#" target="_blank"><img src="https://img.shields.io/badge/Evaluate_with-OOPS! (OntOlogy Pitfall Scanner!)-blue.svg" alt="Evaluate with OOPS!" /></a></dd> --><dt>Cite as:</dt>
<dd>Ana Claudia Sima, Tarcisio Mendes de Farias, Erich Zbinden, Maria Anisimova, Manuel Gil, Heinz Stockinger, Kurt Stockinger, Marc Robinson-Rechavi, Christophe Dessimoz, Enabling semantic queries across federated bioinformatics databases, Database, Volume 2019, 2019, baz106, <a href="https://doi.org/10.1093/database/baz106">https://doi.org/10.1093/database/baz106</a></dd>
<dt>Vocabulary maintained at:</dt>
<dd><a href="https://github.com/biosoda/genex">https://github.com/biosoda/genex</a></dd>
</dl>

<hr/>
</div>
<div class="status">
<div>
<span>accepted</span>
</div>
</div>     <div id="abstract"></div>
<div id="toc"></div>     <div id="introduction"></div>
	  <div id="nstable"></div>
     <div id="overview"></div>
     <div id="description"></div>
     <div id="crossref"></div>
     <div id="references"></div>
<div id="acknowledgments">
<h2 id="ack" class="list">Acknowledgments <span class="backlink"> back to <a href="#toc">ToC</a></span></h2>
<p>
The authors would like to thank <a href="http://www.essepuntato.it/">Silvio Peroni</a> for developing <a href="http://www.essepuntato.it/lode">LODE</a>, a Live OWL Documentation Environment, which is used for representing the Cross Referencing Section of this document and <a href="https://w3id.org/people/dgarijo">Daniel Garijo</a> for developing <a href="https://github.com/dgarijo/Widoco">Widoco</a>, the program used to create the template used in this documentation.</p>
</div>


</div>
</body>
</html>